docs/getting-started/elements/introduction.md

```diff
@@ -0,0 +1,28 @@
+# Elements
+
+Elements provides a component for embedding API reference documentation into any existing CMS, website or React application. Here are some of the features it supports:
+
+- OpenAPI v2 (Swagger), v3.0, v3.1 and beyond
```

Comment:
Maybe not the and beyond part :)

Reply:
Removed it bdd9ff54b1a2434dc52989bae4f17c194597fc8d